frontend-dev/src/components/AllIntegrations/EditCustomFormSubmissionInteg.jsx

```diff
@@ -17,6 +17,10 @@ function EditCustomFormSubmissionInteg({ setSnackbar }) {
   const setFormFields = useSetRecoilState($formFields)
   const [isLoading, setIsLoading] = useState(false)
   const intervalRef = useRef(null)
+  const [skipPrimaryKey, setSkipPrimaryKey] = useState(() => flow?.flow_details?.multi_form && flow?.flow_details?.multi_form?.some(
+    ({ triggered_entity_id, skipPrimaryKey }) => triggered_entity_id === flow?.triggered_entity_id && skipPrimaryKey
```

Comment:
Explain this. why **skipPrimaryKey** ?

Reply:
A primary key acts as the unique identifier for a form (Form ID) and is typically used to distinguish the form by this key. When a trigger is form-based or requires identifying the source of the trigger, the primary key becomes essential. However, when the trigger is not form-based or does not need to identify its origin, the primary key is not necessary.When a primary key is not required, the skipPrimaryKey is used to omit the primary key for such triggers.

For example, when creating a new space, the space has no parent dependency. Therefore, such a trigger does not require a primary key.